tools/js-sdk-release-tools/src/common/npmUtils.ts

```diff
@@ -32,10 +39,51 @@ export function getArtifactName(info: NpmPackageInfo) {
     return `${name}-${version}.tgz`;
 }
 
-export async function tryGetNpmView(packageName: string): Promise<{[id: string]: any} | undefined> {
+export async function tryGetNpmView(packageName: string): Promise<{ [id: string]: any } | undefined> {
     try {
         return await fetch.json(`/${packageName}`);
     } catch (err) {
         return undefined;
     }
+}
+export interface lastStableNpmViewParameter {
+    file: "ApiView" | "CHANGELOG.md";
+    version: string;
+    packageFolderPath: string;
+    packageName: string;
+    sdkRootPath: string;
+    npmPackagePath: string;
+}
+
+export function tryCreateLastStableNpmView(lastStableNpmViewParameter: lastStableNpmViewParameter) {
```

Comment:
I feel like this function can be more general, since this PR is trying to fix current issue, I accept the current fix. But please add TODO comment to refactor it later. Since we can prapare any file from github and the default behavior is to get api view is quite not indicatable from the function name.
```suggestion
// TODO: refactor this function to support praparing files from github in general way
export function tryCreateLastStableNpmView(lastStableNpmViewParameter: lastStableNpmViewParameter) {
```